scripts/install_third_party.sh

```diff
@@ -23,6 +23,7 @@ $PYTHON_CMD scripts/install_third_party.py
 
 # Install third-party node modules needed for the build process.
 $NPM_INSTALL --only=dev
+$NPM_CMD dedupe
```

Comment:
Perhaps explain the use of this line in comment?

Reply:
Thanks for the suggestion @lilithxxx. Addressed in https://github.com/oppia/oppia/pull/6775.